components/arm/fake/fake.go

```diff
@@ -37,9 +37,9 @@ type AttrConfig struct {
 func modelFromName(model, name string) (referenceframe.Model, error) {
 	switch resource.ModelName(model) {
 	case xarm.ModelName6DOF.Name:
```

Comment:
` case xarm.ModelName6DOF.Name, xarm.ModelName7DOF.Name`

Reply:
Good catch, plus Lite also needs to be added